MAUI/CheckBox/Visual-Customization.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: Customization in .NET MAUI CheckBox control | Syncfusion
+description: Learn about Customization support in Syncfusion Essential Studio .NET MAUI CheckBox control, its elements and more.
+platform: Maui
+control: CheckBox
+documentation: ug
+---
+
+# Visual Customization in .NET MAUI CheckBox (SfCheckBox)
+
+## Customizing a shape
+
+The check box shape can be customized using the [`CornerRadius`] property. This property specifies the uniform radius value to the every corner of the check box.
```

Comment:
The check box shape can be customized using the `CornerRadius` property. This property specifies a uniform radius value for every corner of the check box.